website/docs/reference/appsmith-framework/widget-actions/post-message.md

```diff
@@ -55,7 +55,7 @@ windowMessageListener(
 
 | Argument | Description |
 | --- | --- |
-| domain | This is the address of the website that sends the message (`https://mywebsite.com`). The app only listens to messages from the given domain when embedded. If the app is embedded in some other website(`https://myother-website.com`)the callback  won’t be triggered. If an active action is already in place, it won't be overridden and a warning appears in the console. |
+| domain | This is the address of the website that sends the message (`https://mywebsite.com`). The app only listens to messages from the given domain when embedded. If the app is embedded in some other website(`https://myother-website.com`)the callback  won’t be triggered. If an active action is already in place, it won't be overridden and a warning appears in the console. It is not allowed to use `*` as an input for the domain.|
```

Comment:
@eco-monk - Are we only restricting `*` here or other wildcards also, like `?, %`. If yes, then:
@Pranay105 - let's add wildcards and give examples.

Reply:
@jnikhila, User's are likely to try only `*` because that is a general convention to allow all domains.
The other wildcards like `?`/`%` don't mean much in this context, so it's implicitly understood.

cc: @sharat87

Reply:
Thanks @eco-monk.